src/foam/box/sf/SF.js

```diff
@@ -342,9 +344,12 @@ foam.CLASS({
                                     .build();
 
             journalMap_.put(getSimpleFilename(filename), journal);
-            if ( journal.getFileOffset() == journal.getFileSize() ) continue;
+            if ( journal.getFileOffset() == journal.getFileSize() ) {
+              ((DAO) x.get("eventRecordDAO")).put(new EventRecord(getX(), this, "SAF file complete", "Id: " + getId() + ", file: " + getSimpleFilename(filename), LogLevel.INFO, null));
+              continue;
+            }
             if ( journal.getFileOffset() > journal.getFileSize() ) {
-              logger_.warning("Atime of file: " + getSimpleFilename(filename) + " is greater than its filesize");
+              ((DAO) x.get("eventRecordDAO")).put(new EventRecord(getX(), this, "SAF file error", "Id: " + getId() + ", Atime of file: " + getSimpleFilename(filename) + " is greater than its filesize", LogLevel.ERROR, null));
```

Comment:
```suggestion
              ((DAO) x.get("eventRecordDAO")).put(new EventRecord(getX(), this, "SAF file error", getId(), null, "Atime of file: " + getSimpleFilename(filename) + " is greater than its filesize", LogLevel.ERROR, null));
```
use getId as the partner id.  Id is the host is it not?

Reply:
yes, id is host, will update